index.html

```diff
@@ -1429,13 +1421,15 @@ <h2 id="securing-verifiable-credentials">Securing Verifiable Credentials</h2>
         </p>
         <p>The normative statements in <a data-cite="VC-DATA-MODEL-2.0#securing-mechanisms">Securing
             Mechanisms</a> apply to securing
-            <code>application/vc+ld+json</code> and
-            <code>application/vp+ld+json</code>,
-            <code>application/vc+ld+json+sd-jwt</code> and
-            <code>application/vp+ld+json+sd-jwt</code>,
+            <code>application/vc</code> and
+            <code>application/vp</code>,
```

Comment:
I've honestly never really understood this section but it feels like it should only make whatever statement this is making for the media types which this document defines. 
```suggestion
            <code>application/vc</code> and
            <code>application/vp</code>,
```
```suggestion
```